jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/init/MobileUIFactory.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.jdi.light.mobile.elements.init;
+
+import com.epam.jdi.light.elements.init.UIFactory;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.MarkupLocator;
+import com.epam.jdi.light.mobile.elements.base.MobileUIElement;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import static com.epam.jdi.light.driver.WebDriverByUtils.defineLocator;
+
+public class MobileUIFactory extends UIFactory {
+
+    public static MobileUIElement element(@MarkupLocator String locator) {
+        return locator.matches("[A-Z].*]")
+                ? new MobileUIElement().setName(locator)
+                : element(defineLocator(locator));
+    }
+    public static MobileUIElement element(@MarkupLocator By byLocator) {
+        return new MobileUIElement(byLocator);
+    }
+    public static MobileUIElement $(@MarkupLocator String locator) {
```

Comment:
Codacy found an issue: [The static method name '$' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6361799)